### ISSUE TITLE
[clipboard] Copy To Clipboard also Exports to Excel as of version 7.2.x

### ISSUE THREAD

**Issue by dhassouni** [ceolter/ag-grid-enterprise#104]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/ceolter/ag-grid-ng2/blob/master/CONTRIBUTING.md#question
```

**Current behavior**

If the user wants to copy data to a clipboard, using the api command such as `this.api.copySelectedRowsToClipboard(includeHeaders);` as of version 7.2.x, this also triggers an Export to Excel. It also copies the data to the clipboard (which is expected). I experienced this issue using ag-grid-react.

Reverting to 7.1.0 fixed this issue.

**Expected behavior**

When the user copies data to the clipboard using the above api command, it should only copy data to the clipboard without the side effect of also triggering an Excel export.

**Minimal reproduction of the problem with instructions**

Probably easiest to simply update the ag-grid-react-example to use the current version of the relevant libraries.

**What is the motivation / use case for changing the behavior?**

Seems like an unintended side effect in the feature enhancement (Excel Export in v 7.2.x)

**Please tell us about your environment:**
Mac OS X El Capitan v 10.11.6
Webpack
Atom IDE

* **ag-Grid-Enterprise version:** 7.2.3

* **Browser:** Chrome 55.0.2883.95
Didn't try others browsers before reverting to version 7.1.0 to solve this problem.

* **Language:** ES6

**Comment by seanlandsman** [comment]
Hi

This issue has been fixed in 7.2.3 - please upgrade and retry

thanks

**Comment by dhassouni** [comment]
Sean, can you clarify was this ag-grid-enterprise 7.2.3, or was it one of the associated packages? Because as my issue indicated, I had this issue with 7.2.3. This is what I had in package.json:
    "ag-grid": "^7.2.2",
    "ag-grid-enterprise": "^7.2.3",
    "ag-grid-react": "^7.2.0",

**Comment by dhassouni** [comment]
Looking at revision history, I think you meant ag-grid-enterprise 7.2.4, but I can confirm that this issue is resolved there.

**Comment by makinggoodsoftware** [comment]
Hi, yes, I think we got confused in the release numbers, as you say it was fixed in ag-grid-enterprise  7.2.4 not 7.2.3

**Comment by dhassouni** [comment]
Thanks, everyone.
